core/commands/repository/interactors/fetch_repository.py

```diff
@@ -7,23 +8,32 @@
 
 class FetchRepositoryInteractor(BaseInteractor):
     @sync_to_async
+    @sentry_sdk.trace
     def execute(
         self,
         owner: Owner,
         name: str,
         okta_authenticated_accounts: list[int],
         exclude_okta_enforced_repos: bool = True,
-    ):
+    ) -> Repository:
         queryset = Repository.objects.viewable_repos(self.current_owner)
         if exclude_okta_enforced_repos:
             queryset = queryset.exclude_accounts_enforced_okta(
                 okta_authenticated_accounts
             )
 
-        return (
+        # TODO(swatinem): We should find a way to avoid these combinators:
+        # The `with_recent_coverage` combinator is quite expensive.
+        # We only need that in case we want to query these props via graphql:
+        # `coverageAnalytics.{percentCovered,commitSha,hits,misses,lines}`
+        # Similarly, `with_oldest_commit_at` is only needed for `oldestCommitAt`.
```

Comment:
The query generated by the combinators below is this:
https://codecov.sentry.io/insights/backend/database/spans/span/fbbb13620d0da28b/?project=5215654&statsPeriod=30d
As you can see, it is incredibly complex, and also slow.

Reply:
note to self:
https://github.com/mirumee/ariadne/discussions/1116#discussioncomment-6508603
this snippet might help achieve that goal. but by god why that is not part of the standard graphql tooling is beyond me. this looks like fundamental functionality that should just exist out of the box.